### DISCUSSION TITLE
如何去除boost依赖?

### DISCUSSION THREAD

**Discussion by gnleaf** [chenshuo/muduo#606] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

Also specify the exact environment where the issue occurs:

**Comment by chenshuo** [comment]
Try the `cpp17` branch.

**Comment by docterling** [comment]
> Try the `cpp17` branch.

cpp11分支是否有去除boost依赖计划？毕竟cpp17对编译器版本要求高
